.github/workflows/publish.yml

```
@@ -63,8 +63,14 @@ jobs:
           GH_TOKEN: ${{ github.token }}
         working-directory: src/D2L.Bmx/build
         run: |
-          Compress-Archive -Path "$env:FILE_NAME" -DestinationPath "bmx-$env:PLATFORM.zip"
-          gh release upload "$env:RELEASE_TAG" "bmx-$env:PLATFORM.zip"
+          if ($env:PLATFORM -eq 'win') {
+            Compress-Archive -Path "$env:FILE_NAME" -DestinationPath "bmx-$env:PLATFORM.zip"
+            gh release upload "$env:RELEASE_TAG" "bmx-$env:PLATFORM.zip"
```

```suggestion
            Compress-Archive -Path "$env:FILE_NAME" -DestinationPath "bmx-win.zip"
            gh release upload "$env:RELEASE_TAG" "bmx-win.zip"
```

Might as well just do this then if we are okay with this if statement here?

---

and the equivalent for mac

---

might as well